iac-common/src/main/java/org/sonar/iac/common/extension/visitors/InputFileContext.java

```diff
@@ -108,6 +109,10 @@ public void reportAnalysisError(String message, @Nullable TextPointer location)
     error.save();
   }
 
+  private org.sonar.api.batch.fs.TextRange toInputFileRange(TextRange textRange) {
```

Comment:
is TextRange import correct here? or should it be used via the new ones?

Reply:
I would say yes. This method converts the new `TextRange' to the API interface.